src/libopensc/card-starcos.c

```diff
@@ -146,6 +146,8 @@ static int starcos_init(sc_card_t *card)
 		_sc_card_add_rsa_alg(card,1976, flags, 0x10001);
 		_sc_card_add_rsa_alg(card,2048, flags, 0x10001);
 	} else {
+		card->name = "STARCOS SPK 2.3";
```

Comment:
Sorry, I didn't notice that your previous version was actually a fix for a regression introduced with d29bdf9da. So just using fcbcfaa63ed17a01b93070686d435d1c434a08d3 would be preferable

Reply:
I have reset to https://github.com/OpenSC/OpenSC/commit/fcbcfaa63ed17a01b93070686d435d1c434a08d3.